### DISCUSSION TITLE
How I get a text summary use the textgenerator plugin?

### DISCUSSION THREAD

**Discussion by YIRU69** [nhaouari/obsidian-textgenerator-plugin#37] [Q&A] [answered]
Thanks for your share about the fuuny and useful plugin>

And I have a long article or long text and want to know the summary, and could the plugin work it?

**Comment by nhaouari** [comment, marked as answer]
The plugin can work with long articles or texts, but the amount of information that can be summarized will depend on the model used. For example, the text-davinci-003 model can handle up to 4,000 tokens, while the text-curie-001 and text-babbage-001 models are limited to 2,048 tokens each. You can find more information about the models in the following link: [https://beta.openai.com/docs/models/overview](https://beta.openai.com/docs/models/overview). 

To try it under your text, write a new line `summary:`, then a new empty line and Generate Text (Ctrl + J). The documentation of the plugin is available in [https://text-gen.com/](https://text-gen.com/).
